libs/datasets/timeseries.py

```diff
@@ -478,9 +482,17 @@ def from_combined_dataframe(
         return multiregion_timeseries
 
     @staticmethod
-    def from_csv(path_or_buf: Union[pathlib.Path, TextIO]) -> "MultiRegionTimeseriesDataset":
+    def from_csv(
+        path_or_buf: Union[pathlib.Path, TextIO], provenance: Optional[pd.Series] = None,
```

Comment:
I was curious in what case we would be loading a CSV but have already-parsed provenance to go along with it...  It turns out we don't really.  But in tests, we load the provenance from a (string) CSV and then pass the _parsed_ version (as `pd.Series`) into from_csv() along with the _unparsed_ primary CSV (as just a `TextIO`).

This seems like a bit arbitrary / inconsistent.  Why not have `provenance` be `Optional[TextIO]` and always do the CSV parsing inside this method?  Or for full symmetry between the primary CSV and the provenance CSV, you could do `Optional[Union[pathlib.Path, TextIO]]` but that's overly flexible since we never need to pass in a Path right now.  🤷

Reply:
Yes, that sucked. Fixed with https://github.com/covid-projections/covid-data-model/pull/757